.github/workflows/auto-merge.yml

```
@@ -0,0 +1,19 @@
+name: Dependabot auto-merge
+on: pull_request_target
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  enable-auto-merge:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'jhipster/generator-jhipster' && github.event.pull_request.user.login == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1.3.0
+      - name: Enable auto-merge for Dependabot PRs
+        if: ${{steps.dependabot-metadata.outputs.update-type == 'version-update:semver-minor'}}
```

I think it's better to exclude "major" only, and allow auto merge for patch and minor, instead of only minor:

```suggestion
        if: ${{ steps.dependabot-metadata.outputs.update-type != 'version-update:semver-major' }}
```